packages/blockchain-wallet-v4-frontend/src/data/components/sendBch/sagas.ts

```diff
@@ -88,10 +88,10 @@ export default ({
         payment = yield payment.from(defaultIndex, ADDRESS_TYPES.ACCOUNT)
         if (to) payment = yield payment.to(to)
         if (amount && amount.coin) {
-          const satAmount = Exchange.convertBchToBch({
+          const satAmount = Exchange.convertCoinToCoin({
             value: amount.coin,
-            fromUnit: 'BCH',
-            toUnit: 'SAT'
+            baseToStandard: false,
```

Comment:
can we just default this to false in function so we dont have to pass?

Reply:
or default it to whatever case is used more commingly

Reply:
sure